lib/platforms/knex/0.11/client.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var Promise = require('bluebird');
+var _ = require('lodash');
+var tracker = require('../../../tracker');
+
+var connection = {
+  id : 'mockedConnection',
+};
+
+function mockClient(client) {
+  client.driverName = 'mocked';
+
+  client.acquireConnection = function () {
+    return {
+      completed: connection,
+      abort: function () { },
+    };
```

Comment:
In 0.11, `completed` should resolve to the connection and we're already in a promise at that point so no need for `Promise.resolve(connection)`. The `abort` function seems like it should no-op, since it's used to abort resource acquires in pool2.